cli/mender-artifact/debugfs.go

```diff
@@ -104,53 +105,100 @@ func debugfsReplaceFile(imageFile, newFile, image string) (err error) {
 	executeCommand(cmd, image)
 	// Write to the partition
 	cmd = fmt.Sprintf("cd %s\nwrite %s %s\nclose", filepath.Dir(imageFile), newFile, filepath.Base(imageFile))
-	return executeCommand(cmd, image)
+	_, err = executeCommand(cmd, image)
+	return err
+}
+
+func debugfsRemoveFileOrDir(imageFile, image string, recursive bool) (err error) {
+	// Check that the file or directory exists.
+	cmd := fmt.Sprintf("cd %s", filepath.Dir(imageFile))
+	if _, err = executeCommand(cmd, image); err != nil {
+		return err
+	}
+	isDir := filepath.Dir(imageFile) == filepath.Clean(imageFile)
+	var deleteCmd string
+	if isDir {
+		return debugfsRemoveDir(imageFile, image, recursive)
+	} else {
+		deleteCmd = "rm"
```

Comment:
Maybe this should be out of the else

My reasoning: all of it should be in the else (set deleteCmd, set cmd, executeCommand) or none of it.